fe1-web/network/NetworkConnection.ts

```diff
@@ -0,0 +1,155 @@
+import { w3cwebsocket as W3CWebSocket } from 'websocket';
+import {
+  JsonRpcRequest, JsonRpcResponse, ProtocolError, UNDEFINED_ID,
+} from 'model/network';
+import { NetworkError } from './NetworkError';
+
+type ResponseHandler = (message: JsonRpcResponse) => void;
+
+const DEFAULT_RESPONSE_HANDLER: ResponseHandler = (message: JsonRpcResponse) => {
+  if (message.error === undefined) {
+    // default handler does nothing which is useful for some messages
+  } else {
+    console.error(
+      'A negative network error was received:\n'
+      + `\t- error code : ${message.error.code}`
+      + `\t- description : ${message.error.description}`,
+    );
+  }
```

Comment:
I don't think we need much here, to be honest, other than just logging something like "No response handler was provided to manage message: xxxx"